services/eclipse-marketplace/eclipse-marketplace-downloads.tester.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
```

Comment:
Were there any changes in the tests, or were they just moved?

Reply:
The five existing tests we copy-pasted as is in the different tester classes. Additionally, I added one "not found" test case for each implementation.